composer.json

```diff
@@ -125,6 +125,7 @@
         "amphp/amp": "^2.5",
         "amphp/http-client": "^4.2.1",
         "amphp/http-tunnel": "^1.0",
+        "amphp/parallel": "^1.x-dev",
```

Comment:
why version 1.x-dev? 
Version 2 has been released for over a year. I think we should use that for this new feature.

Reply:
It's just that on the root composer, it's not compatible with amphp/amp version 2.5 (only from v3), but at the Symfony Messenger component level, the version of amphp/parallel is indeed set to v2.

Reply:
Can't we just remove this line for now in the root composer file then?